### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by dgdavid**
This PRs apply some UX improvements recommended by the UI/UX SUSE Team. Namely, 

* Use a CTA for the `Add route` button

   | Before | After |
   |-|-|
   |  Screenshot_2020-12-18_at_11 41 04 |  Screenshot_2020-12-24 Networking - ytm 10 0 0 204 |

* Reallocate the interface/connection actions 

  | Before | After |
  | - | - |
  | Screenshot_2020-12-18_at_12 09 08 | Screen Shot 2020-12-24 at 13 10 02 |

### CODE REVIEW

**Comment by teclator** [comment]
In general, LGTM

File: src/components/InterfaceActions.test.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            );
+
+            const actionsButton = screen.getByLabelText('Actions');
+            userEvent.click(actionsButton);
+
+            expect(screen.getByText('Reset')).toBeInTheDocument();
+        });
+    });
+
+    describe('when interface is disable', () => {
```
Comment (imobachgs) [inline src/components/InterfaceActions.test.js:69]: typo: *disabled*.

File: src/components/InterfaceActions.test.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+import { createConnection } from '../lib/model/connections';
+import { createAddressConfig } from '../lib/model/address';
+import InterfaceActions from "./InterfaceActions";
+
+const connection = createConnection({ id: 1, name: 'eth0' });
+
+jest.mock('../lib/NetworkClient');
+
+describe('InterfaceActions', () => {
+    describe('when interface is enable', () => {
```
Comment (imobachgs) [inline src/components/InterfaceActions.test.js:37]: typo: *enabled*.

**Review by imobachgs — APPROVED** [review]
Just minor typos. Otherwise, LGTM.

**Pull request merged**